packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1579,6 +1607,189 @@ test.serial('updateGranuleStatusToQueued() throws error if record does not exist
   );
 });
 
+test.serial('updateGranuleStatusToQueued() does not update DynamoDB or Elasticsearch granule if writing to PostgreSQL fails', async (t) => {
+  const {
+    collectionCumulusId,
+    esGranulesClient,
+    esClient,
+    granule,
+    granuleId,
+    granuleModel,
+    granulePgModel,
+    knex,
+  } = t.context;
+
+  const testGranulePgModel = {
+    get: () => Promise.resolve(granule),
+    update: () => {
+      throw new Error('Granules Postgres error');
+    },
+  };
+
+  await writeGranuleFromApi({ ...granule }, knex, esClient, 'Create');
+  const dynamoRecord = await granuleModel.get({ granuleId });
+  const postgresRecord = await granulePgModel.get(
+    knex,
+    { granule_id: granuleId, collection_cumulus_id: collectionCumulusId }
+  );
+  const esRecord = await esGranulesClient.get(granuleId, granule.collectionId);
+
+  t.is(dynamoRecord.status, 'completed');
+  t.is(postgresRecord.status, 'completed');
+  t.is(esRecord.status, 'completed');
+  t.truthy(dynamoRecord.execution);
+
+  await t.throwsAsync(updateGranuleStatusToQueued({
+    granule: dynamoRecord,
+    knex,
+    granulePgModel: testGranulePgModel,
+  }));
```

Comment:
```suggestion
  await t.throwsAsync(updateGranuleStatusToQueued({
    granule: dynamoRecord,
    knex,
    granulePgModel: testGranulePgModel,
  }), { error: 'Granules Postgres error' });
```

Let's check that the expected error is what is thrown for all of these tests, so that we know the test is passing for the reason that we expect

Reply:
Great idea! [`76fa2a6` (#2593)](https://github.com/nasa/cumulus/pull/2593/commits/76fa2a67db5b64ba73caa6661327787ba93525aa)